__tests__/app.test.js

```diff
@@ -45,45 +46,43 @@ describe("/api/nonsense", () => {
 });
 
 describe("/api", () => {
-    test("GET - status 200 - resonds with JSon object of all availiable endpoints", () => {
-        return request(app)
-            .get("/api")
-            .expect(200)
-            .then((result) => {
-              expect(result.body.endpoints).toEqual(endpoints);
-              
-            });
-    });
-
-})    
+  test("GET - status 200 - resonds with JSon object of all availiable endpoints", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((result) => {
+        expect(result.body.endpoints).toEqual(endpoints);
```

Comment:
This test is not testing anything. You're asserting the thing you've imported is the same as the file you've imported.

Reply:
I had different test previously that checked the contents, however, in my feedback I was told i only needed to check it against the file, so that I didn't need to updated my tests every ticket. Am I supposed to check it the contents for each endpoint?

![image](https://github.com/SaskiaWells/NC_news/assets/122098877/bc582e18-eb96-400f-a4fc-206024b71a71)